pom.xml

```diff
@@ -363,6 +370,41 @@
                     <tagNameFormat>v@{project.version}</tagNameFormat>
                 </configuration>
             </plugin>
+            <plugin>
+                <!--
+                http://revapi.org/modules/revapi-maven-plugin/examples/comparing-specific-artifacts.html
+
+                mvn clean package -Darguments="-DskipTests" -DskipTests -Pjavadoc-jdk-8u121
+
+                mvn revapi:report-aggregate -Darguments="-DskipTests" -DskipTests -Pjavadoc-jdk-8u121
+
+                open target/site/revapi-aggregate-report.html
+                -->
+                <groupId>org.revapi</groupId>
+                <artifactId>revapi-maven-plugin</artifactId>
+                <version>0.9.1</version>
+                <configuration>
+                    <newArtifacts>
+                        <artifact>${project.groupId}:${project.artifactId}:${project.version}</artifact>
+                    </newArtifacts>
+                </configuration>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>report</goal>
+                            <goal>report-aggregate</goal>
```

Comment:
I tried to add the `report-aggregate` within `install` or `site` phases, but it didn't work either. For now, I removed the problematic goal.